mkosi/state.py

```diff
@@ -61,6 +62,10 @@ def workdir(self) -> Path:
     def staging(self) -> Path:
         return self.workspace / "staging"
 
+    @property
+    def pkgmngr(self) -> Path:
+        return self.workspace / "pkgmngr"
+
```

Comment:
You introduced this in the wrong commit, this should be done as the first commit